runtime/rasdump/j9dmp.tdf

```diff
@@ -29,6 +29,7 @@ TraceEvent=Trc_dump_J9VMDllMain_Event2 NoEnv Overhead=1 Level=1 Template="Dump c
 TraceEvent=Trc_dump_reportDumpStart_Event1 NoEnv Overhead=1 Level=1 Template="JVM Requesting %s Dump using filename=%s"
 TraceEvent=Trc_dump_reportDumpError_Event2 NoEnv Overhead=1 Level=1 Template="Error in %s Dump: %s"
 TraceEvent=Trc_dump_reportDumpEnd_Event2 NoEnv Overhead=1 Level=1 Template="%s Dump written to filename=%s"
+TraceEvent=Trc_dump_reportDumpEnd_Event3 NoEnv Overhead=1 Level=1 Template="%s Dump finished writing"
```

Comment:
New trace events must be added at the end (to retain ids of existing events).

Reply:
I guess since we'll be stick with the existing messages (Ie, what we had before). Is it still nessesary to create a new trace event, or can we use the same event as priting to a file (Dump using filename...)

Reply:
Agreed (see my next comment).